app/controllers/agents_controller.rb

```diff
@@ -7,6 +7,22 @@ def index
     set_table_sort sorts: %w[name created_at last_check_at last_event_at last_receive_at], default: { created_at: :desc }
 
     @agents = current_user.agents.preload(:scenarios, :controllers).reorder(table_sort).page(params[:page])
+    @shared_agents = Agent.shared - @agents
```

Comment:
Ah good point yes, that's actually a stray line that's no longer used anywhere - removed